luskan/src/main/java/org/opencadc/luskan/TempStorageManager.java

```diff
@@ -169,12 +174,14 @@ public URL put(Throwable t, TableWriter writer) throws IOException {
         }
         // TODO: store requested content-type with file so that 
         // TempStorageGetAction can set content-type header correctly
+
```

Comment:
Why can't this also be scheduled for deletion after QueryJobManager.MAX_DESTRUCTION like the inline content? I think the method results in approximately the same amount of time. The solution is simple so should be implemented as simply as possible.

Reply:
Updated to use the QueryJobManager.MAX_DESTRUCTION for all deletions.